tests/dualtor/test_ipinip.py

```diff
@@ -115,7 +115,7 @@ def stop_garp(ptfhost):
 
     if is_t0_mocked_dualtor(tbinfo):        # noqa F405
         request.getfixturevalue('apply_active_state_to_orchagent')
-        time.sleep(30)
+        wait(60, 'wait for config push/mux state change on mocked dualtor')
```

Comment:
what's the reason to change sleep to wait?

Reply:
Hi @kevinskwang , changed it just for consistency(and a msg) after observing that we are using wait instead of sleep at other places.

Reply:
Hi @kevinskwang , do we want to change it back to sleep?